jina/helper.py

```diff
@@ -1000,7 +1000,17 @@ def _update_policy():
             )
 
 
-def get_or_reuse_loop_with_info():
+class ManagedLoop(object):
```

Comment:
u will need to redefine `__getattr__` to forward it to `_loop` to make it transparent, or access the private loop to start it